docs/how-to/what-is-rocblas.rst

```diff
@@ -319,6 +319,14 @@ In addition to the above API, rocBLAS also provides an environment variable ``RO
 Bitwise Reproducibility
 -----------------------
 
+Bitwise reproducible results in rocBLAS can be obtained under the following conditions:
```

Comment:
This new content shouldn't be in this gray background:
![image](https://github.com/user-attachments/assets/de2d8bce-2312-40e1-8bbc-3c3ef6516d12)